src/opers.c

```diff
@@ -449,48 +449,46 @@ static Int IsSubsetFlagsCalls2;
 *F  UncheckedIS_SUBSET_FLAGS( <flags1>, <flags2> ) subset test with
 *F                                                         no safety check
 */
-static Obj UncheckedIS_SUBSET_FLAGS (
-    Obj                 flags1,
-    Obj                 flags2 )
+static Obj UncheckedIS_SUBSET_FLAGS(Obj flags1, Obj flags2)
 {
-    Int                 len1;
-    Int                 len2;
-    UInt *              ptr1;
-    UInt *              ptr2;
-    Int                 i;
-    Obj                 trues;
+    Int    len1;
+    Int    len2;
+    UInt * ptr1;
+    UInt * ptr2;
+    Int    i;
+    Obj    trues;
 
-    /* do the real work                                                    */
+/* do the real work                                                    */
 #ifdef COUNT_OPERS
     IsSubsetFlagsCalls++;
 #endif
 
     /* first check the trues                                               */
     trues = TRUES_FLAGS(flags2);
-    if ( trues != 0 ) {
+    if (trues != 0) {
         len2 = LEN_PLIST(trues);
-        if ( TRUES_FLAGS(flags1) != 0 ) {
-            if ( LEN_PLIST(TRUES_FLAGS(flags1)) < len2 ) {
+        if (TRUES_FLAGS(flags1) != 0) {
+            if (LEN_PLIST(TRUES_FLAGS(flags1)) < len2) {
 #ifdef COUNT_OPERS
                 IsSubsetFlagsCalls1++;
 #endif
                 return False;
             }
         }
-        if ( len2 == 0 ) {
+        if (len2 == 0) {
             return True;
         }
-        if ( len2 < 3 ) {
+        if (len2 < 16) {
```

Comment:
Why the change from 3 to 16?

Reply:
Experimentally 16 seems to be roughly optimal for modern CPUs and compilers. The gain from pushing it up from 3 to say 10 is quite noticeable. After that the differences are smaller.

Reply:
Apart from CPU and compiler changes, the total number of flags has increased a lot over the years.

Reply:
OK, thanks for the clarification. It's just that this change is kind of hidden amidst the huge refactoring. Might be good idea to mention it in the commit message?